Controller/CRUDController.php

```diff
@@ -221,7 +221,8 @@ public function deleteAction($id)
     /**
      * Edit action.
      *
-     * @param int|string|null $id Optional ID to use instead of globally provided default.
+     * @param int|string|null $id Optional ID representing the main admin subject.
```

Comment:
> In another way, I think we should instead reintroduce request parameter directly on action (And maybe param fetcher) instead of playing each time with the request.

Agreed, but this is not BC, and is long term.

For short term, I'd say revert be82d59, and let's not deprecate `$id` until what we have decied what we want in the future.